routing/routes_builder/routes_builder.cpp

```diff
@@ -138,8 +138,11 @@ RoutesBuilder::Result RoutesBuilder::ProcessTask(Params const & params)
 
 std::future<RoutesBuilder::Result> RoutesBuilder::ProcessTaskAsync(Params const & params)
 {
-  Processor processor(m_numMwmIds, m_dataSourcesStorage, m_cpg, m_cig);
```

Comment:
Why MSVC compiler didn't like it?

Reply:
Ditto MSVC STL bug

Reply:
Please, add a comment here, because code is a bit strange now